state/postgresql/metadata.go

```diff
@@ -57,6 +58,20 @@ func (m *postgresMetadataStruct) InitWithMetadata(meta state.Metadata) error {
 		return err
 	}
 
+	if m.TableName != "" {
```

Comment:
This is making the postgres cert test fail.

The reason is that some DBs have different rules for valid table names. In the case of Postgres (and probably others), `.` is a valid character if you want to specify the schema name.

Reply:
Then I will just remove this validation for table names again and leave it the way it was. In 1.11 it can then be done properly.  What do you think? Not likely someone can edit the yaml but doesn't have the credentials to the DB anyway.

I certainly won't be writing custom validation code for each DB here.